lib/tai/advisors/factories/shared_venues_and_shared_products.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Tai.Advisors.Factories.SharedVenuesAndSharedProducts do
+  @moduledoc """
+  Advisory factory for sharing all subscribed products. Advisor name falls back to `group.id`.
```

Comment:
`Advisory` -> `Advisor`

How do you feel about ✂️ `Advisor name falls back to 'group.id'.` as there is no other possibility. It's always the `group.id`?

Reply:
do you mean in the `moduledoc` or in `opts`?

Reply:
Within the `moduledoc`.

Upon reviewing this again I feel like the advisor id should be something like `default` or `main` as the advisor name is generated as a union between group id & advisor id.